pageserver/src/layered_repository.rs

```diff
@@ -1435,6 +1471,14 @@ impl LayeredTimeline {
         let metadata_path =
             LayeredRepository::save_metadata(self.conf, self.timelineid, self.tenantid, &metadata)?;
         if let Some(relish_uploader) = &self.relish_uploader {
+            let _local_timeline = LocalTimeline {
```

Comment:
Again, here goes the download instead, no separate metadata file upload is happening: https://github.com/zenithdb/zenith/pull/686/files#diff-2dee9987054422607c1ab7369e102927d8477dea6d75c4c7f3891e3d65b15d1eR1471-R1479

Leaf as is to not to drag in the relish_storage related changes.